### PR TITLE
tw/updated readme

### PR DESCRIPTION
**Pull request by TusharW4ni**
Revamped the readme file to look like Taz's example. Added Project Setup section. Kept the stuff from the previous semester but commented it out. In the first commit to this branch, I had tried to style the toggles to look like H3. It worked in VS Code however in github it didn't render out properly so in the subsequent commit I just took that part out.

### CODE REVIEW

**Comment by TusharW4ni** [comment]
Please let me know, if you want any changes.
Thank you.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Getting Started with Create React App
+<!---# Getting Started with Create React App
```
Comment (MStolen) [inline README.md]: You can just remove this whole block comment. It will all be saved in the version history, so there is no reason to keep it.

**Review by MStolen — CHANGES_REQUESTED** [review]
I would just like to see the big comment at the top removed and this will be a great update.

**Comment by TusharW4ni** [comment]
Alright, removed the commented text 👍.

**Review by MStolen — APPROVED** [review]

**Pull request merged**